lib/beaker/host/unix/exec.rb

```diff
@@ -12,16 +12,17 @@ module Unix::Exec
   #
   # Will throw an exception RebootFailure if it fails
   def reboot(wait_time=10, max_connection_tries=9, uptime_retries=18)
+    require 'time'
+
     begin
-      original_uptime = exec(Beaker::Command.new("uptime"))
-      original_uptime_str = parse_uptime original_uptime.stdout
```

Comment:
I'll have to remember to yank out `parse_uptime` once this is merged, since I'm pretty sure it and `uptime_time` are both dead code now.

Reply:
I wasn't sure if nobody else was using them so I left them out there.

Reply:
I'm pretty sure they were only added as part of the rework we did to make reboot use `uptime`, so I *think* they should be safe to remove. https://github.com/puppetlabs/beaker/pull/1619/files#diff-adecec78dcc971d847136c54308d7766R37